app/services/list_tasks.rb

```diff
@@ -0,0 +1,20 @@
+class ListTasks < ListCollection
+
+  attr_defaultable :task_repository, -> { Task }
+  attr_defaultable :result_serializer, -> { V1::TaskSerializer }
+
+  def initialize project_id: nil
+    super()
+    @project_id = project_id
+  end
+
+  def collection_type
+    :tasks
+  end
+
+  def collection
+    @tasks ||= task_repository.where(project_id: project_id).where.not(state: -1)
```

Comment:
Might be worth considering making a scope or class method on `Task` to represent the concept of getting all Tasks `where.not(state: -1)`. It seems like this `ListTasks` needs to have intimate knowledge of the underlying storage of state in the database. Perhaps more than might be proper.